tests/test_logitech.py

```diff
@@ -11,7 +11,7 @@ def test_keyboard_base_basic_check(keyboard_base):
 
     assert str(keyboard_base) == 'KeyboardManager: 160x43'
     logitech_repr = repr(keyboard_base)
-    data = ('parser', 'ProtocolParser', 'plane_name', 'plane_detected', 'lcdbutton_pressed', 'buttons',
```

Comment:
Looks like I hadn't reverted this change in the test. Doesn't affect anything so can be readded in a later release.

Reply:
I will add in master